libs/core/src/lib/carousel/carousel.component.scss

```diff
@@ -17,3 +17,18 @@ fd-carousel:focus {
         }
     }
 }
+
+@media (max-width: 21.25rem) {
```

Comment:
please move it to fundamental-styles

Reply:
+1 to Yury's comment. All the css should be moved in Fund-styles.